webserver.py

```diff
@@ -125,13 +133,20 @@ def index():
     # Build the template variables
     unnamed_devices = filter(lambda x: x['name'] is None, devices)
     named_devices = filter(lambda x: x['name'] is not None, devices)
+    
+    notes = get_last_note()
+    notes = [{
+        'note': d['note'],
+        'pretty_date': pretty_date( get_diff_from_now( unix_to_bst(d['unixdate']))),
+    } for d in notes]
 
     return render_template('index.html',
                            devices=unnamed_devices,
                            named_devices=named_devices,
                            harmony_domain=HARMONY_DOMAIN,
-						   city=CITY,
-						   country=COUNTRY)
+                           city=CITY,
+                           country=COUNTRY,
+                           note=notes[0])
```

Comment:
this will throw an index error if there are no notes

Reply:
Added an if statement to return an empty note otherwise